fastlane/Fastfile

```diff
@@ -16,7 +16,7 @@ require 'securerandom'
 ENV["FASTLANE_XCODEBUILD_SETTINGS_TIMEOUT"] = "600"
 
 default_platform(:ios)
-xcode_select("/Applications/Xcode_15.0.app")
+xcode_select("/Applications/Xcode_15.0.1.app")
```

Comment:
@DylanRJ why have we had to update this? I'm guessing, but is this to do with the version on the gitrunner, which has changed since the last time we merged a PR?

Reply:
@elishkaflint this is something I haven't changed back since testing the different versions earlier.
[Xcode 15 release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-15-release-notes) states a known issue "Executing Unit/UI tests from Xcode on the iOS Simulator takes an extended time to launch on first run of the suite."
This is said to have been resolved in [Xcode 15.0.1 release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-15_0_1-release-notes)
I haven't reverted this back to Xcode 15 to test but happy to

Reply:
Oh I see! You did mention this - thanks!